.eslintcr.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [ 1, {
+      "js": "always", "json": "always"
+    }]
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
+  ]
+}
```

Comment:
- Great Job for adding es lint,👍👍 however  your es lint is not running because you didn’t update your `linters.yml` file to run eslint, so please update your `linters.yml` based on this [js linter](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml)👍👌